app/__init__.py

```diff
@@ -1,15 +1,25 @@
 from flask import Flask
 from flask_bcrypt import Bcrypt
 from flask_sqlalchemy import SQLAlchemy
+from azure.cosmos import errors, CosmosClient
+
 
 import os
 
 APP_DIR = os.path.abspath(os.path.dirname(__file__))
-STATIC_FOLDER = os.path.join(APP_DIR, '../static/build/static') # Where your webpack build output folder is
-TEMPLATE_FOLDER = os.path.join(APP_DIR, '../static/build') # Where your index.html file is located
+STATIC_FOLDER = os.path.join(
+    APP_DIR, "../static/build/static"
+)  # Where your webpack build output folder is
+TEMPLATE_FOLDER = os.path.join(
+    APP_DIR, "../static/build"
+)  # Where your index.html file is located
 
 app = Flask(__name__, static_folder=STATIC_FOLDER, template_folder=TEMPLATE_FOLDER)
-app.config.from_object('app.config.ProductionConfig')
+app.config.from_object("app.config.ProductionConfig")
+
+key = app.config["SECRET_KEY"]
+uri = app.config["COSMOS_DB_URI"]
+
+client = CosmosClient(uri, credential=key)
```

Comment:
I'm not crazy about having the cosmos client as a global variable for the app.   I would like all the "cosmos" stuff to be encapsulated so the app is only coupled to that encapsulation versus directly to cosmos.

Reply:
Done, I think. Added a db.py in /utils and kept cosmos things there. Import the db in models.py to use. I thought this way, if in the future, we want to use other db, we can keep all db things there and just import the appropriate dbs to use in models.py or other scripts.